### PR TITLE
[PoC] GUI: Migrate BIP70 merchant info to mapValue["to"]

### PR DESCRIPTION
**Pull request by luke-jr**
The idea being to save this information in a usable format before we remove BIP70 entirely...

Thoughts?

### CODE REVIEW

File: src/interfaces/wallet.cpp [hunk 1]
```diff
@@ -354,6 +354,13 @@ class WalletImpl : public Wallet
         }
         return {};
     }
+    bool writeWalletTxValues(const uint256& txid, std::map<std::string, std::string> new_map)
```
Comment (practicalswift) [inline src/interfaces/wallet.cpp]: Should be marked `override`?

**Comment by jonasschnelli** [comment]
Unsure about this... what is the long term use case for the stored data? Would it eventually be more straight forward to give steps how to keep the BIP70 data in a release notes (once completely removed)?
Or would altering "dumpwallet" with an option to dump the BIP70 stuff make sense?

**Comment by luke-jr** [comment]
The BIP70 stuff is currently stored in a protobuf-encoded format IIRC, so accessing it post-removal or even from RPC would be invasive.

**Comment by laanwj** [comment]
I think this makes some sense in context of deprecating BIP70. The data should remain in the wallet (for legal/historical purposes) but will effectively become inaccessible. 
Migrating it to another field makes sure that something of the historical metadata is visible even after BIP70 parsing is removed.

**Pull request closed without merging**